python/Ganga/Core/GangaRepository/Registry.py

```diff
@@ -788,7 +768,8 @@ def _flush(self, _objs=None):
             for obj_id in obj_ids:
                 self.unlock_transaction(obj_id)
 
-            self._lock.release()
+            for obj in objs:
+                obj._releaseParentLock()
```

Comment:
Ahh, I see. `_releaseParentLock` doesn't exist so this will just raise an exception instead of double-releasing?